src/main/scala/org/broadinstitute/dsde/agora/server/model/AgoraApiJsonSupport.scala

```diff
@@ -167,7 +174,19 @@ object AgoraApiJsonSupport extends DefaultJsonProtocol {
 
   implicit val AccessControlPairFormat = jsonFormat3(EntityAccessControl)
 
-  implicit object AgoraStatusFormat extends RootJsonFormat[AgoraStatus] {
+  implicit val MethodRepoMethodFormat = jsonFormat3(MethodRepoMethod)
+
+  // AttributeString is just a case class with one String in it, but for some reason jsonFormat1 is not sufficient
+  // https://stackoverflow.com/a/28530361/818054
+  implicit val AttributeStringFormat = jsonFormat1(AttributeString)
+  implicit object attributeStringJsonFormat extends RootJsonFormat[AttributeString] {
+    override def read(value: JsValue) = AttributeString(value.convertTo[String])
+    override def write(f: AttributeString) = JsString(f.value)
+  }
+
+  implicit val MethodConfigurationFormat = jsonFormat10(MethodConfiguration)
+
```

Comment:
the rawls model library already has json formats defined for `MethodRepoMethod`, `MethodConfiguration`, and `AttributeString`.

Look for:
`WorkspaceModel.MethodStoreMethodFormat`
`WorkspaceModel.MethodConfigurationFormat`
`JsonSupport.AttributeStringFormat`